controllers/operatorpolicy_status.go

```diff
@@ -394,7 +415,29 @@ func missingWantedCond(kind string) metav1.Condition {
 	}
 }
 
-// createdCond returns a Compliant condition, with a Reason like'____Created',
+// missingNotWantedCond returns a Compliant condition with a Reason like '____Excluded'
+// and a Message like 'the ____ checked by the mustnothave policy was not found'
+func missingNotWantedCond(kind string) metav1.Condition {
+	return metav1.Condition{
+		Type:    condType(kind),
+		Status:  metav1.ConditionTrue,
+		Reason:  kind + "Excluded",
```

Comment:
What about this?
```suggestion
		Reason:  kind + "NotPresent",
```

Reply:
Updated this. "Included" is also now "Present" and the deleted and kept conditions are updated so that their messages are shorter and avoid the "checked by the mustnothave" language.